src/DevAdventCalendarCompetition/DevAdventCalendarCompetition/Controllers/TestController.cs

```diff
@@ -38,16 +44,21 @@ public ActionResult Index(int testNumber)
         [HttpPost]
         public ActionResult Index(int testNumber, string answer = "")
         {
-            if (answer is null)
+            if (answer == null)
```

Comment:
why do you make that change?

Reply:
Everywhere we use "==", so why to break the convention?

Reply:
now I understand :) ok then